prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -268,6 +268,12 @@ void efa_rdm_ope_try_fill_desc(struct efa_rdm_ope *ope, int mr_iov_start, uint64
 		if (ope->desc[i])
 			continue;
 
+		if (OFI_UNLIKELY(ope->ep->base_ep.domain->mr_local))
+			EFA_WARN(FI_LOG_EP_CTRL,
+				 "No valid desc is provided, not complied with FI_MR_LOCAL."
+				 "buf: %p len: %ld access: %lx\n",
```

Comment:
```suggestion
				 "No valid desc is provided, not compliant with FI_MR_LOCAL. "
				 "buf: %p len: %ld access: %#lx\n",
```

`#` automatically prepends the conversion with `0x`/`0X` - just now noticing the many instances of `%x`/`%lx` in our logs without it.